documentation/schema.md

```diff
@@ -92,7 +95,27 @@ The value of a fill_in step can be a single field, or a list of hashes defining
     It's very useful where max length is only enforced server-side.
 - `wysiwyg` (optional): This field will be present if a field is a wysiwyg, such as ckeditor or tinymce. They cannot be filled in the same way as a typical textbox.
 
-**A note on CAPTCHAs**
+### remove
+Removes DOM element from page.
+- `selector`: Selector for the DOM element to remove
+
+### recaptcha
+
+Clicks the reCAPTCHA checkbox to open the challenge, then fetchs either the image or audio challenge (depending on configuration).
```

Comment:
Can you add something about audio selectors being optional if only image is desired?